buildtools/tinygo/Dockerfile

```diff
@@ -10,11 +13,8 @@ RUN curl -L https://go.dev/dl/go1.19.1.linux-${TARGETARCH:-amd64}.tar.gz | tar -
 ENV PATH /go/bin:/root/go/bin:$PATH
 ENV GOROOT /go
 
-RUN apt-get install -y libclang-14-dev wabt binaryen git
+RUN apt-get install -y libclang-14-dev wabt binaryen
```

Comment:
Shall we run `rm -rf /var/lib/apt/lists/*` after the install to reduce image size?